test/src/main/java/io/strimzi/test/extensions/StrimziExtension.java

```diff
@@ -285,8 +284,7 @@ private boolean isWrongClusterType(ExtensionContext context) {
      */
     private boolean isWrongClusterType(AnnotatedElement element) {
```

Comment:
I wasn't to make this work via mvn test execution - `mvn verify -pl systemtest -P systemtests -Djava.net.preferIPv4Stack=true -DtrimStackTrace=false -DTEST_CLUSTER=test -Dit.test=OpenShiftTemplatesST -DfailIfNoTests=false`  where `test` is my openshift context for cluster on different machine. According the output it seems that extension is trying to find context with name `minishift`:

```[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.01 s <<< FAILURE! - in io.strimzi.systemtest.OpenShiftTemplatesST
[ERROR] io.strimzi.systemtest.OpenShiftTemplatesST  Time elapsed: 0.009 s  <<< ERROR!
org.junit.jupiter.engine.execution.ConditionEvaluationException: 
Failed to evaluate condition [io.strimzi.test.extensions.StrimziExtension]: `oc config get-contexts minishift` got status code 1 and stderr:
------
error: context minishift not found

------
and stdout:
------
CURRENT   NAME      CLUSTER   AUTHINFO   NAMESPACE

------
```

@chonton  Does it work for you? I am able to run tests only from Idea, but it skip `@OpenshiftOnly` annotation evaluation

Reply:
TEST_CLUSTER is an environment variable, not a java system property.  Try
```TEST_CLUSTER=test mvn verify -pl systemtest -P systemtests -Djava.net.preferIPv4Stack=true -DtrimStackTrace=false -Dit.test=OpenShiftTemplatesST -DfailIfNoTests=false```

Reply:
Oh I see, seems to be fine now :)  thanks